src/simplewallet/simplewallet.cpp

```diff
@@ -3648,6 +3648,21 @@ static std::string get_human_readable_timestamp(uint64_t ts)
   return std::string(buffer);
 }
 //----------------------------------------------------------------------------------------------------
+static std::string get_human_readable_timespan(uint64_t ts)
+{
+  if (ts < 60)
+    return std::to_string(ts) + tr(" seconds");
+  if (ts < 3600)
+    return std::to_string((int)(ts / 60)) + tr(" minutes");
```

Comment:
C++ does not convert divisions into floating point. The types are `uint64 / int` which will result in a `uint64` result with a floor truncation on the division. So the current cast is unnecessary but harmless.